package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cache-buster",
-  "version": "0.1.5",
```

Comment:
Can u remove version increment line? Because I'm gonna add one more feature with this PR.